cli/command_snapshot_migrate.go

```diff
@@ -44,13 +45,19 @@ func (c *commandSnapshotMigrate) setup(svc advancedAppServices, parent commandPa
 	c.out.setup(svc)
 }
 
+//nolint:funlen
 func (c *commandSnapshotMigrate) run(ctx context.Context, destRepo repo.RepositoryWriter) error {
 	sourceRepo, err := c.openSourceRepo(ctx)
 	if err != nil {
 		return errors.Wrap(err, "can't open source repository")
 	}
 
-	defer sourceRepo.Close(ctx) //nolint:errcheck
+	defer func() {
+		//nolint:errcheck
+		destRepo.Close(ctx)
```

Comment:
We should let `destRepo` be closed by `app.go:App::maybeRepositoryAction`. WDYT ?